lib/autocop/version.rb

```diff
@@ -1,3 +1,3 @@
 module Autocop
-  VERSION = '0.1.4'.freeze
+  VERSION = '0.1.5'.freeze
```

Comment:
Don't change the version in pull requests. Also why your build is failing is because of this being changed and the `Gemfile.lock` not being updated.